File: framework/SampleSDKDemoObjC/SampleSDKDemoObjC/ViewController.m
//
//  ViewController.m
//  SampleSDKDemoObjC
//

#import "ViewController.h"
#import <SampleSDK/SampleSDK.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // testing RoboVM framework
    SampleSDK* sdk = SampleSDKInstance();
    NSLog(@"RoboVM version %@", [sdk roboVmVersion]);

    Calculator* calc = [sdk newCalculator];
    [calc add:123];
    [calc sub:23];
    NSLog(@"%d", [calc result]);
    
    [[sdk webServer] start];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: framework/SampleSDKDemoSwift/SampleSDKDemoSwift/ViewController.swift
//
//  ViewController.swift
//  SampleSDKDemoSwift
//

import UIKit
import SampleSDK

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

        // testing RoboVM framework
        var sdk:SampleSDK!;
        sdk = SampleSDKInstance();
        NSLog("RoboVM version %@", sdk.roboVmVersion());

        var calc:Calculator;
        calc = sdk.newCalculator();
        calc.add(123);
        calc.sub(23);
        NSLog("%d",calc.result());
        
        sdk.webServer().start();
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

